sdk/eventhub/event-hubs/rollup.base.config.js

```diff
@@ -108,16 +108,9 @@ export function browserConfig(test = false) {
         }
       ),
 
-      // dotenv, path, and os don't work in the browser, so replace it with a no-op function
+      // dotenv doesn't work in the browser, so replace it with a no-op function
       shim({
-        fs: `export default {}`,
-        dotenv: `export function config() { }`,
-        os: `
-          export function arch() { return "javascript" }
-          export function type() { return "Browser" }
-          export function release() { return typeof navigator === 'undefined' ? '' : navigator.appVersion }
-        `,
-        path: `export default {}`
+        dotenv: `export function config() { }`
```

Comment:
We need this only for test code right? Can we add a `if (test)` check?

Reply:
Sure! Though the browser rollup bundle is only used for tests since we don't publish it :smile:

Reply:
hehe

Reply:
Then we can let it be

Reply:
Oh woops, I made the change before seeing your response, so it is only added for tests now.